modules/network-observability-packet-drops.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-packet-drops_{context}"]
+= Working with packet drops
+Packet loss occurs when one or more packets of network flow data fail to reach their destination. You can track these drops by enabling the `spec.agent.ebpf.enablePktDrop` and `spec.agent.ebpf.privileged` specifications in the `FlowCollector`.
++
+[IMPORTANT]
+====
+Cpu and memory usage increases when this feature is enabled.
+====
+
+.Prerequisites
+* Access to an {product-title} cluster with version 4.13. 
+* Kernel supported by Red Hat Enterprise Linux (RHEL) 9.2.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource so that `spec.agent.ebpf.enablePktDrop` and `spec.agent.ebpf.privileged` specifications to `true`. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-dns_{context}"]
+.Configure `FlowCollector` for packet drops
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+namespace: netobserv
```

Comment:
there is spec here, the YAML should look like

apiVersion: flows.netobserv.io/v1alpha1
kind: FlowCollector
metadata:
     name: cluster
spec:
   namespace: netobserv
  deploymentModel: DIRECT
  agent:
    type: EBPF
    ebpf:
      enablePktDrop: true
      privileged: true

Reply:
Addressed.